full_snapshot.proto

```diff
@@ -141,6 +142,16 @@ message DatabaseInformation {
   bool collected_local_catalog_data = 11;
 }
 
+message DatabaseStatistic {
+  int32 database_idx = 1;
+
+  uint32 transactions_per_sec = 2; // Transactions per second for this database
+  uint32 min_xid = 3; // Oldest transaction ID for this database
+  uint32 min_mxid = 4; // Oldest multixact ID for this database
+  uint32 xid_age = 5; // Oldest transaction age for this database
+  uint32 mxid_age = 6; // Oldest multixact age for this database
```

Comment:
[note for reviewer]
This is a database level txid/mxid information. txid/mxid info is the same as relation stats, and in addition to that, the database stats contains TPS (transactions per sec) per database.

In theory, especially with the non-busy table/db, `min_xid` and `min_mxid` won't change much. However, `xid_age` and `mxid_age` would change more often as it's doing the diff between the current transaction.
Potentially we could do the age calculation not on the collector side in the future (as we do collect `current_xact_id` below), but let's collect age info also as an initial version.

Reply:
Re: transactions per sec: I think it might be best if we keep these as two separate counters (xact_commit and xact_rollback), see also my note on the collector PR (https://github.com/pganalyze/collector/pull/342/files#r1049132746).

That way we have flexibility to add a "rolled back transactions" graph in the future, and wouldn't need any collector changes to do so - the space here is cheap, since its just one number for each database.